source/irfs/irf_components/index.rst

```diff
@@ -118,8 +123,7 @@ PSFs must be normalised to integrate to total probability 1, i.e.
 
 .. math::
 
-    \int_{0}^{\infty} 2 \pi r dP/dr(r) dr = 1, where dP/dr = 2 \pi r dP/d\Omega
-
+    \int_{0}^{\infty} dP/d\Omega(r) d\Omega = 1\,.
```

Comment:
@lmohrmann - Omega is the solid angle, I don't think the integral as written is correct.
either you put a generic `\int dP/dOmega dOmega` with the understanding that it's a 2D integral, or you have to write an integral over the 1D radial `r`, no?

Reply:
True, I removed the bounds.